### PR TITLE
chandan.html, style.css & Photo Added!

### PR DESCRIPTION
**Pull request by chandancodes**

### CODE REVIEW

File: Tasks/Chandan/chandan.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+
+<head>
+    <title>Chandan Singh Codes</title>
+    <link rel="stylesheet" href="./style.css">
```
Comment (sharananurag998) [inline Tasks/Chandan/chandan.html]: Please link chandan.css here instead of style.css

**Comment by chandancodes** [comment]
Can you please tell me what to be done? 
Why needs revision?

**Comment by chandancodes** [comment]
Done!

File: Tasks/Chandan/chandan.html (hunk 1 above)
Comment (chandancodes) [reply Tasks/Chandan/chandan.html]: Please check I just did a new commit

**Comment by chandancodes** [comment]
Can you please check again?

**Pull request merged**